.rubocop.yml

```diff
@@ -212,3 +224,327 @@ Style/HashTransformKeys:
 
 Style/HashTransformValues:
   Enabled: true
+
```

Comment:
Generic comment on the file, rather than on that specific line: I'm actually not too sure what was the rationale for including or not a rule in there.
Since there are a lot of rules I'd expect to include them only if we change the default, but I don't know for sure if we've done that with the past updates.

Did you include all the rules that were added/available or did you pick just those for which we needed to change the default configuration?
(and do you have an opinion on if we should include everything or not?)

Reply:
I included all of the new ones that were available, even if it meant pasting their default config.
This has two reasons:
(a) If a rule is not explicitly included, RuboCop reminds you after every update that "Hey, these cool new rules exist and could be added to your config" by printing some message to stdout. That message in turn messes with overcommit.
(b) I feel that explicitly listing stuff gives us a better overview about what features exist in the first place, without having to dig through the documentation. Especially since a lot of the checks have clear and concise names anyways.